src/notation/qml/MuseScore/NotationScene/internal/CapoPopup.qml

```diff
@@ -33,7 +33,7 @@ StyledPopupView {
     property alias navigationOrderStart: capoSettingsNavPanel.order
     readonly property alias navigationOrderEnd: capoSettingsNavPanel.order
 
-    property QtObject model: capoModel
+    readonly property alias model: capoModel
```

Comment:
Do we still need this property at all, now that the `model.canOpen` call no longer happens in QML but in C++?